### PR TITLE
Bottom bar

### PR DESCRIPTION
**Pull request by liviavinci**

### CODE REVIEW

**Comment by liviavinci** [comment]
screen shot 2018-03-01 at 5 28 56 pm
screen shot 2018-03-01 at 5 28 47 pm

File: src/languageProvider/entries/en-US.js [hunk 1]
```diff
@@ -4,9 +4,10 @@ import enMessages from '../locales/en_US.json';
 
 const EnLang = {
   messages: {
-    'cornerclock.online': 'Online',
-    'cornerclock.offline': 'Offline',
-    'cornerclock.block': 'Current Block',
+    'bottombar.online': 'Online',
+    'bottombar.offline': 'Offline',
+    'bottombar.blockNum': 'Current Block Number',
+    'bottombar.blockTime': 'Current Block Time',
```
Comment (dwalintukan) [inline src/languageProvider/entries/en-US.js]: use camelCase for the component name too: ie. `bottomBar.blockTime`

File: src/languageProvider/entries/zh-Hans-CN.js [hunk 2]
```diff
@@ -4,9 +4,10 @@ import zhMessages from '../locales/zh-Hans.json';
 
 const ZhLan = {
   messages: {
-    'cornerclock.online': '在线',
-    'cornerclock.offline': '离线',
-    'cornerclock.block': '当前区块数',
+    'bottombar.online': '在线',
+    'bottombar.offline': '离线',
+    'bottombar.blockNum': '当前区块数',
+    'bottombar.blockTime': '当前区块时间',
```
Comment (dwalintukan) [inline src/languageProvider/entries/zh-Hans-CN.js]: camelCase

File: src/components/BottomBar/index.js [hunk 3]
```diff
@@ -29,41 +38,62 @@ class BottomBar extends React.PureComponent {
   }
 
   render() {
-    const { syncBlockNum, syncBlockTime } = this.props;
+    const { classes, syncBlockNum, syncBlockTime } = this.props;
+
+    return (
+      syncBlockTime && syncBlockTime ?
```
Comment (dwalintukan) [inline src/components/BottomBar/index.js]: syncBlockNum && syncBlockTime?

File: src/components/BottomBar/index.js [hunk 4]
```diff
@@ -29,41 +38,62 @@ class BottomBar extends React.PureComponent {
...
+        <Paper className={classes.bottomBarWrapper}>
+          <Grid container>
+            {this.renderNetworkConnection()}
+            {this.renderBlockInfo()}
+          </Grid>
+        </Paper> : null
```
Comment (dwalintukan) [inline src/components/BottomBar/index.js]: do you think we should still show this bar even if the we cant get syncBlockTime/num? and maybe show error text: `Error getting block info`?

Also, the network connection text should be visible always

**Comment by dwalintukan** [comment]
looks good!

File: src/components/BottomBar/index.js (hunk 3 above)
Comment (liviavinci) [reply src/components/BottomBar/index.js]: this is to hide the bottom bar if data is not loaded

File: src/components/BottomBar/index.js (hunk 4 above)
Comment (liviavinci) [reply src/components/BottomBar/index.js]: Let's hide the block number if it's not loaded. It's not nice to display error to users directly. And there is the loading phase so...

**Pull request merged**